### PR TITLE
docs(README): add README.md

### LINKED ISSUE #10: Add README
## Description
As a contributor,
I need to understand what the project is about and how to setup,
so that I can contribute to it.

I think a first version of the README, should include at least:
- description of the app (mission, objective, etc)
- how to setup, dependencies, version of Android
- communication channel (e.g.: Zulip link and stream dedicated)

### Update [Required]
- [ ] Add a first version of the README

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
30 min

### PR DESCRIPTION
**Pull request by kbhardwaj123**
### Description
Add Readme for the repository

Fixes #10  Add Readme

### Type of Change:
- Documentation

### Checklist:
**Delete irrelevant options.**

- [x] Issue for this PR exists and has been approved
- [x] My PR follows the style guidelines of this project

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by kbhardwaj123** [comment]
@annabauza waiting for suggestions :)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Branches
+
+* **master** This contains the code which has been deployed into production.
+
+ * **develop** This contains the latest code. All the contributing PRs must be sent to this branch. When we want to release the next version of the app, this branch is merged into the `master` branch.
+
+## Contact
+
+Have a query, feel free to reach out to our team @ on our Slack channel [#STEM-diverse-tv](https://anitab-org.zulipchat.com/login/#narrow/stream/225705-STEM-diverse-tv) where all the discussion on this project are carried out.
```
Comment (isabelcosta) [inline README.md]: @kbhardwaj123 Its not Slack but Zulip :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+In order to setup the project locally just follow the steps described below:
+
+ - Fork the project repository then clone your fork to your system.
+ - Open the project using Android Studio
+ - You have successfully set up the application.
+
+## Contributing
+
+Please read our [Contributing guidelines](https://github.com/anitab-org/stem-diverse-android-tv/blob/master/.github/CONTRIBUTING.md), [Code of Conduct](https://github.com/anitab-org/stem-diverse-android-tv/blob/master/.github/CODE_OF_CONDUCT.md) and [Reporting Guidelines](http://systers.io/reporting-guidelines)
```
Comment (isabelcosta) [inline README.md]: If you use relative links then these links will function properly on a contributor's fork and other branches different than `master` @kbhardwaj123

**Comment by kbhardwaj123** [comment]
@isabelcosta all changes are in place:)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Branches
+
+* **master** This contains the code which has been deployed into production.
+
+ * **develop** This contains the latest code. All the contributing PRs must be sent to this branch. When we want to release the next version of the app, this branch is merged into the `master` branch.
+
+## Contact
+
+Have a query, feel free to reach out to our team @ on our Zulip channel [#STEM-diverse-tv](https://anitab-org.zulipchat.com/login/#narrow/stream/225705-STEM-diverse-tv) where all the discussion on this project are carried out.
```
Comment (isabelcosta) [inline README.md]: the reach out to @ makes little sense, since you mention no specific tag.

Comment (kbhardwaj123) [reply README.md]: @isabelcosta oh sorry I have removed that now :)

**Pull request merged**